README.md

```diff
@@ -49,8 +51,28 @@ Make sure everything installed correctly by running the unit tests through
 This should also solve the [example problem](ifopt_core/include/ifopt/ex_problem.h) with your installed solvers. 
 If you have [IPOPT] installed and linked correctly, this should also execute the 
 binary [ifopt_ipopt-test](ifopt_ipopt/test/ex_test_ipopt.cc). 
-    
-     
+
+### CMake
+* Clone this repo and configure (``cd`` to desired directory before doing the following)
+
+      $ git clone https://github.com/ethz-adrl/ifopt.git
+      $ mkdir build ; cd build
+      $ cmake -DCATKIN_BUILD=OFF -DCMAKE_INSTALL_PREFIX=path_to_build_dir ..
```

Comment:
Best case this is just `cmake ..` and the system automatically detects that it should build without catkin and where the files should be installed. Possibly its even better to avoid system-wide installation and just keep all files local in the `build` folder.